apps/app/src/Screens/Types/Stack.ts

```diff
@@ -0,0 +1,5 @@
+export type RootStackParamList = {
+    Home: undefined;
+    Friends: { name: string };
+    Quests: { name: string };
+  }
```

Comment:
**question (non-blocking):** The property names (`Home`, `Friends`, `Quests`) should be the same names as the screens, right? Is there a way to enforce that on the type level (so you can't forget one or mistype one)?

Reply:
Like we discuss [using enum to list the screens](https://github.com/dbrekelmans/questline-galaxy/pull/4/commits/af2b27952026254ebf5dd836644ef78d9a12f5da)